.github/workflows/tests-integration.yaml

```diff
@@ -23,6 +23,22 @@ jobs:
           python-version: 3.13
           activate-environment: true
 
+      - name: Wake Up HuggingFace Endpoint
+        run: |
+          curl "https://y0okp71n85ezo5nr.us-east-1.aws.endpoints.huggingface.cloud/v1/chat/completions" \
+          -X POST \
+          -H "Authorization: Bearer ${{ secrets.HF_TOKEN }}" \
+          -H "Content-Type: application/json" \
+          -d '{
+              "model": "tgi",
+              "messages": [
+                  {
+                      "role": "user",
+                      "content": "Wake up!"
+                  }
+              ]
+          }'
```

Comment:
Can we move this into something that I can easily run locally too? Maybe into the scripts folder?

Also, should we change the behavior so that it will wait til the server responds 200 before finishing?